doc/src/stdlib/test.md

```diff
@@ -169,15 +170,15 @@ As calculations on floating-point values can be imprecise, you can perform appro
 checks using either `@test a ≈ b` (where `≈`, typed via tab completion of `\approx`, is the
 [`isapprox()`](@ref) function) or use [`isapprox()`](@ref) directly.
 
-```julia-repl
+```jldoctest
 julia> @test 1 ≈ 0.999999999
+Test Passed
 
 julia> @test 1 ≈ 0.999999
-ERROR: test failed: 1 isapprox 0.999999
- in expression: 1 ≈ 0.999999
- in error at error.jl:21
- in default_handler at test.jl:30
- in do_test at test.jl:53
+Test Failed
+  Expression: 1 ≈ 0.999999
+   Evaluated: 1 isapprox 0.999999
```

Comment:
Thanks, I messed up when running the doctests (they didnt actually get run) so I will rebase master and make sure everything passes.